calicoctl/calico_ctl/node.py

```diff
@@ -87,6 +87,9 @@
 CALICO_NETWORKING_ENV = "CALICO_NETWORKING"
```

Comment:
Well, I think we chose to do it this was essentially because we weren't sure if we wanted to make it a documented feature at the time (we obviously do now!)

It's also a nicely generic way of passing configuration through to a command - pretty much any orchestrator will let you configure the environment in which you're running a command.

That said, it feels like `calicoctl node --backend=[native|bird|gobgp|...]` or similar is a pretty nice UX.  Either way, if we do choose expose it as a command line option it would also be really convenient to have it as an environment variable as well.